### PR TITLE
[control-plane-manager] Let streaming audit log to stdout

### PR DESCRIPTION
**Pull request by shvgn**
## Description

- Add option to stream audit log to stdout
- Add scenarios to FAQ
  - How to use stdout
  - How to omit default deckhouse polkicies

## Why do we need it, and what problem does it solve?

Rotating logs in file is not always convenient, especially with statically predefined rotation parametes

## Changelog entries

```changes
section: control-plane-manager
type: feature
summary: Added option to stream audit log to stdout
---
section: control-plane-manager
type: feature
summary: Added option to change audit log files location
```

### CODE REVIEW

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -146,21 +146,45 @@ title: "Управление control plane: FAQ"
...
+
+### Как вывести аудит-лог в стандартный вывод вместо файлов?
+
+Установите параметр `apiserver.auditPolicy.output` в значение `stdout`.
+
+```yaml
+controlPlaneManager: |
+  apiserver:
+    auditPolicyEnabled: true
+    auditPolicy:
```
Comment (shvgn) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:175]: `auditPolicy` probably should be named `auditLog`

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 2]
```diff
@@ -150,6 +150,29 @@ The control-plane-manager saves backups to `/etc/kubernetes/deckhouse/backup`. T
     ```bash
     kubectl -n kube-system create secret generic audit-policy --from-file=./audit-policy.yaml
     ```
+### How to omit deckhouse built-in policy rules?
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:153]:
```suggestion
### How to omit Deckhouse built-in policy rules?
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -92,6 +92,15 @@ properties:
       basicAuditPolicyEnabled:
         description: |
           Применять базовые политики аудита Deckhouse.
+      auditPolicy:
+        description: Параметры аудита
+        required:
+          - output
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:98]:
```suggestion
        description: Параметры аудита
```

File: modules/040-control-plane-manager/docs/FAQ.md (hunk 2 above)
Comment (shvgn) [reply modules/040-control-plane-manager/docs/FAQ.md]: fixed

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml (hunk 3 above)
Comment (shvgn) [reply modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml]: fixed

**Review by nabokihms — APPROVED** [review]

**Pull request merged**